tmail-backend/mailbox/encrypted/cassandra/src/main/java/com/linagora/tmail/encrypted/cassandra/table/CassandraEncryptedEmailStoreModule.scala

```diff
@@ -0,0 +1,26 @@
+package com.linagora.tmail.encrypted.cassandra.table
+
+import com.datastax.driver.core.DataType.{cboolean, cint, frozenMap, text, timeuuid}
+import com.datastax.driver.core.schemabuilder.Create.Options
+import com.datastax.driver.core.schemabuilder.SchemaBuilder.KeyCaching.ALL
+import com.datastax.driver.core.schemabuilder.SchemaBuilder.rows
+import com.datastax.driver.core.schemabuilder.{Create, SchemaBuilder}
+import org.apache.james.backends.cassandra.components.CassandraModule
+import org.apache.james.backends.cassandra.utils.CassandraConstants.DEFAULT_CACHED_ROW_PER_PARTITION
+
+object CassandraEncryptedEmailStoreModule {
+  val MODULE: CassandraModule = CassandraModule.table(EncryptedEmailTable.TABLE_NAME)
+    .comment("Holds content of encrypted email")
+    .options((options: Options) => options
+      .caching(ALL, rows(DEFAULT_CACHED_ROW_PER_PARTITION))
+      .compressionOptions(SchemaBuilder.lz4.withChunkLengthInKb(8)))
```

Comment:
I'm not sure configuring compression make sense here (encrypted content can likely be large...)

Reply:
Honestly, I am not sure too. Maybe I will spend time digging into these options soon.

Reply:
If we are not sure please let the defaults here: we can tune it later ;-)